osgp/protocol-adapter-iec61850/osgp-protocol-adapter-iec61850/src/main/java/org/opensmartgridplatform/adapter/protocol/iec61850/infra/networking/services/commands/Iec61850EnableReportingCommand.java

```diff
@@ -76,7 +75,22 @@ public void enableReportingOnDeviceWithoutUsingSequenceNumber(final Iec61850Clie
 
         reportListener.setSqNum(0);
         reporting.writeBoolean(SubDataAttribute.ENABLE_REPORTING, true);
-        LOGGER.info("Allowing device {} to send reports containing events", deviceConnection.getDeviceIdentification());
+        LOGGER.info("Allowing device {} to send buffered reports containing events",
+                deviceConnection.getDeviceIdentification());
+    }
+
+    public void enableUnbufferedReportingOnDeviceWithoutUsingSequenceNumber(final Iec61850Client iec61850Client,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "iec61850Client". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1172)